packages/office-ui-fabric-react/src/components/Button/MessageBarButton/MessageBarButton.styles.ts

```diff
@@ -2,16 +2,27 @@ import { IButtonStyles } from '../Button.types';
 import { ITheme, concatStyleSets } from '../../../Styling';
 import { memoizeFunction } from '../../../Utilities';
 import { getStyles as getBaseButtonStyles } from '../BaseButton.styles';
+import { getFocusStyle } from '@uifabric/styling';
```

Comment:
This import can and should be done in line 2 import.

Reply:
Thanks for catching :) The import was added by vscode, I believe.